actors/csharp/sdk/README.md

```diff
@@ -34,15 +34,15 @@ Run the `SmartDevice.Service`, which will start service itself and the Dapr side
 <!-- STEP
 name: Run actor service
 expected_stdout_lines:
-  - "Request finished HTTP/1.1 GET http://127.0.0.1:5001/healthz - - - 200"
+  - "Request finished HTTP/1.1 GET http://127.0.0.1:5001/healthz - 200"
```

Comment:
This is the line of output from healthz that changed, causing our tests to fail.  by changing the expected out we fix it back up again.